packages/main/src/Popup.hbs

```diff
@@ -6,6 +6,8 @@
 	aria-label="{{_ariaLabel}}"
 	aria-labelledby="{{_ariaLabelledBy}}"
 	dir="{{dir}}"
+	tabindex="-1"
+	@keydown={{_onRootFocusOut}}
```

Comment:
Please call this method `_onkeydown`. For event handlers it is preferable to call the function based on what event is handled, instead of on what the function does. Later, if you need to implement some other kind of KH, it would have to be in the same method, as it is the one bound to the event, and then the name `_onRootFocusOut` would no longer be entirely correct and then you would have to rename it to `_onkeydown` or something similar, so better to do it now.

Reply:
Done